src/baklava/CustomNode.vue

```diff
@@ -137,22 +137,38 @@ export default class CustomNode extends Components.Node {
 }
 </script>
 
-<style scoped>
+<style scoped lang="scss">
   #arrow-button {
     position: absolute;
     top: 5px;
     right: 0;
     padding-right: 10px;
   }
-  #header {
-    color: white;
-    padding: 0.4em 0.75em;
-    border-radius: 4px 4px 0 0;
-    font-size: 16px;
-    font-weight: 800;
-    text-align: center;
-  }
+
   #content {
     background: var(--dark-grey);
   }
+
+  .node {
+    /*font-family: Roboto, serif;*/
```

Comment:
Left for when the font is applied globally, and last styling of nodes is to be done.